lib/process-data-availability.js

```diff
@@ -212,22 +197,33 @@ const getAvailabilityAggregatesByResourceTemplate = () => {
  * @param {Object} availabilityReport the RESO availability report JSON to process.
  * @returns a JSON availability report with the appropriate roll-ups and aggregates.
  */
-const processDataAvailability = async (availabilityReport) => {
+const processDataAvailability = async availabilityReport => {
   //iterate over each field and lookup and compute their availabilities
   const { resources, fields, lookups, lookupValues, ...rest } = availabilityReport;
 
   const transformed = getAvailabilityTemplate();
   const standardFieldCache = createStandardFieldCache(standardMetadata.fields);
 
   const resourceCounts = {};
-  resources.forEach(
-    (resource) =>
-      (resourceCounts[resource.resourceName] = resource.numRecordsFetched)
-  );
+  const nestedResourcesCount = {};
+  resources.forEach(resource => (resourceCounts[resource.resourceName] = resource.numRecordsFetched));
+  resources.forEach(resource => {
+    if (resource.expansions) {
+      resource.expansions.forEach(expansion => {
+        if (!nestedResourcesCount[resource.resourceName]) {
+          nestedResourcesCount[resource.resourceName] = {};
+        }
+        nestedResourcesCount[resource.resourceName][expansion.resourceName] = expansion.numRecordsFetched;
+      });
+    }
+  });
 
+  const lookupValuesWithParentResourceName = lookupValues.filter(l => l.parentResourceName);
+  const lookupValuesWithoutParentResourceName = lookupValues.filter(l => !l.parentResourceName);
   const processedFields = [],
     processedLookupValues = [],
-    lookupValueCache = createLookupValueCache(lookupValues);
+    lookupValueCacheWithoutParentResourceName = createLookupValueCache(lookupValuesWithoutParentResourceName),
```

Comment:
The terminology here could be improved. 

Lookup values without a parent resource name are just lookup values. 

Lookup values with a parent resource name are expanded lookup values.